jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/PatternMatcher.java

```diff
@@ -14,90 +14,20 @@
 package org.eclipse.jetty.util;
 
 import java.net.URI;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.regex.Pattern;
 
+// TODO: replace with UriPatternPredicate or ResourceUriPatternPredicate to avoid Resource -> URI -> Resource flows.
```

Comment:
If you're introducing these new predicates, I don't see the point in retaining the PatternMatcher class.

Reply:
If nothing else is using it, i'll remove it.

Reply:
Removed in commit 1066ae04f37ac0b585ad8705826cadade90e9e4c